forc-explore/src/main.rs

```diff
@@ -47,11 +49,15 @@ struct GitHubReleaseAsset {
 const REPO_RELEASES_URL: &str = "https://api.github.com/repos/FuelLabs/block-explorer-v2/releases";
 
 #[tokio::main]
-async fn main() -> Result<()> {
+async fn main() {
     let app = App::parse();
-    match app.subcmd {
+    let result = match app.subcmd {
         Some(Subcommand::Clean) => clean(),
         None => run(app).await,
+    };
+    if let Err(err) = result {
+        error!("Error: {:?}", err);
+        std::process::exit(1);
     }
```

Comment:
Are you able to try @Voxelot's suggestion [here](https://github.com/FuelLabs/sway/issues/884#issuecomment-1125264218) as an alternative to manually checking the error like this?

Reply:
Yep, I actually started with that approach but while using that approach the errors are automatically structured with a trailing `error=` (for errors). To remove that field in the output I need to create a custom layer. [This](https://github.com/FuelLabs/sway/pull/1575/commits/945a4b15e780011099b8b6e58092eaa7a828ab0c) commit uses `#[insturment]` and a custom layer and visitor for removing that extra `error=` field.